ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -371,7 +376,18 @@ objects:
         export TF_LOG=DEBUG
         export TF_LOG_PATH=/tmp/artifacts/terraform.log
 
-        export TF_VAR_libvirt_master_memory=16384
+        # Support legacy libvirt master memory override
+        if [[ -n "${LEGACY_TF_MEMORY_OVERRIDE}" ]]; then
+            export TF_VAR_libvirt_master_memory=16384
```

Comment:
Why have a switch? Why not set both env var and the value inside manifests?

Reply:
I never tested what would happen in a 4.2 build if you set the manifests as well as the env var. I'll try that now.

My intention behind handling this this way was to keep the 4.2 jobs clear of as many hacks as possible. I feel that using sed to swap out the previous memory assignment from a known hardcoded value is functional, but also bad. If the libvirt installer was ever updated to bump the default master memory up or down - or a something else were added to the manifests that used 7168 for some other purpose, this would cause 4.3+ jobs to break because of this hack. By including this hack in 4.2, that would break as well.

The longterm goal for the upstream community is to re-add support for the env vars for terraform. I doubt this would get backported to 4.3 & 4.4, but if we decided to do that, we could completely eliminate this hack entirely.

One better way to do would be to use a tool to parse the yaml and explicitly override just the key we're looking for, but that would involve added a dependency to the libvirt installation image, which is not something I can do in the short timeline I have to get this into CI,